remme/rpc_api/transaction.py

```diff
@@ -204,16 +206,20 @@ def _get_proto_validator(current_handler, tr_payload_pb):
         raise KeyNotFound(f'Transactions with ids "{ids}" not found')
 
 
-@validate_params(ProtoForm, ignore_fields=('ids', 'start', 'limit', 'head', 'reverse'))
+@validate_params(ListBatchesForm)
 async def list_batches(request):
     ids = request.params.get('ids')
     start = request.params.get('start')
     limit = request.params.get('limit')
     head = request.params.get('head')
     reverse = request.params.get('reverse')
 
-    return await account_client.list_batches(ids, start, limit, head, reverse)
-
+    try:
+        return await account_client.list_batches(ids, start, limit, head, reverse)
+    except (KeyNotFound, ClientException):
+        raise KeyNotFound(f'Resource not found.')
```

Comment:
```suggestion
        raise KeyNotFound(f'List of batches not found.')
```

Reply:
https://remmeio.atlassian.net/browse/REM-1126

Reply:
@Art17, this is default error, we need to change it.